internal/mage/sdk/python.go

```diff
@@ -136,6 +136,23 @@ func (t Python) Publish(ctx context.Context, tag string) error {
 	return err
 }
 
+// Bump the Python SDK's Engine dependency
+func (t Python) Bump(ctx context.Context, version string) error {
+	lines := []string{version}
+	if strings.Contains(version, "@") {
+		lines = strings.SplitN(version, "@", 2)
+		lines[0] += "@"
+	}
+
+	versionFile := "# Code generated by dagger. DO NOT EDIT.\n\nENGINE_IMAGE_REF = (\n"
+	for _, l := range lines {
+		versionFile += fmt.Sprintf("    %q\n", l)
+	}
+	versionFile += ")\n"
+
+	return os.WriteFile("sdk/python/src/dagger/connectors/engine_version.py", []byte(versionFile), 0600)
+}
+
```

Comment:
You can simplify by putting in the same line. Just add the `# flake8: noqa` comment on the second line like in the generated files from codegen.

Reply:
I prefer to make these a commit, instead of a comment. I expanded on this here: https://github.com/dagger/dagger/pull/3648#issuecomment-1302300696 - it even includes a diagram!

Reply:
Feel free to improve in a follow-up PR:
- https://github.com/dagger/dagger/pull/3786/files#diff-146c6b8ae73ba9457d30356f062c0dce2099eaf95a84aa2cba28f68d32241b76
- https://github.com/dagger/dagger/pull/3786/commits/3b8abf8622e47d8ace4dbb174b9a8ee546ad19ed